mlflow/models/output_parsers.py

```diff
@@ -0,0 +1,60 @@
+from dataclasses import asdict
+from typing import Any, Dict
+
+from mlflow.models.rag_signatures import (
+    ChainCompletionChoice,
+    ChatCompletionResponse,
+    Message,
+    StringResponse,
+)
+from mlflow.utils.annotations import experimental
+
+try:
+    from langchain_core.output_parsers.transform import BaseTransformOutputParser
+
+    @experimental
+    class ChatCompletionsOutputParser(BaseTransformOutputParser[Dict[str, Any]]):
+        """
+        OutputParser that wraps the string output into a dictionary representation of an MLflow
+        ChatCompletionResponse
```

Comment:
Can we update this to be a link to the MLflow ChatCompletionResponse object in documentation?